src/App.js

```diff
@@ -1,16 +1,67 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+import './components/TodoComponents/Todo.css';
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      todos: [
+      ],
+      task: '',
+    };
+  }
+
+  addTodo = e => {
+    e.preventDefault();
+    const todos = this.state.todos;
+    todos.push({ task: this.state.task, completed: false, id: Date.now() });
+    this.setState({ todos, task: '' });
```

Comment:
It's good practice not to mutate the state directly with `.push()`
Consider the following code instead:
```
addTodo = e => {
    e.preventDefault();
    this.setState({
      todos: [
        ...this.state.todos,
        {task: this.state.task, completed: false, id: Date.now()}
      ],
      task: ""
    });
  };
```